rsocket/framing/ScheduledFrameTransport.h

```diff
@@ -34,6 +34,15 @@ class ScheduledFrameTransport : public FrameTransport,
   void closeWithError(folly::exception_wrapper ex) override;
 
  private:
+  DuplexConnection* getConnection() override {
+    DLOG(FATAL)
+        << "ScheduledFrameTransport doesn't support getConnection method, "
+            "because it can create safe usage issues when EventBase of the "
+            "transport and the StateMachine is not the same.";
```

Comment:
"StateMachine" here sounds like `RSocketStateMachine`, but  it's actually the "state machine evb" in the `RSocketClient`.  The `RSocketStateMachine` will always use the same thread as the transport.

I'd replace "StateMachine" with "RSocketClient", I feel like that's a little clearer.